src/util/__tests__/importRoam.ts

```diff
@@ -0,0 +1,180 @@
+// constants
+import {
+  ROOT_TOKEN,
+} from '../../constants'
+
+// util
+import {
+  hashContext,
+  hashThought,
+} from '../../util'
+
+// selectors
+import {
+  exportContext,
+} from '../../selectors'
+import { importROAM } from '../importROAM'
+import { State, initialState } from '../initialState'
+import { RoamNode } from 'roam'
+import { exportedWithoutRoot } from '../../test-helpers/exportWithoutRoot'
+import { SimplePath, Timestamp } from '../../types'
+
+const RANKED_ROOT = [{ value: ROOT_TOKEN, rank: 0 }]
+const testState: State = {
+  ...initialState(),
+  thoughts: {
+    contextIndex: {
+      [hashContext([ROOT_TOKEN])]: {
+        context: [ROOT_TOKEN],
+        children: [],
+        lastUpdated: '11-10-2020' as Timestamp
```

Comment:
`11-10-2020` is not a valid `Timestamp`. A `Timestamp` is an ISO string, i.e. `new Date().toISOString()`. I recommend importing the `timestamp` function to create new `Timestamps`.

Reply:
Okay, I'll do that